scipy/stats/tests/test_stats.py

```diff
@@ -768,79 +768,88 @@ def test_gh_8111(self):
         assert_allclose([res1, res2, res3], expected)
 
 
-def test_spearmanr():
-    # Cross-check with R:
-    # cor.test(c(1,2,3,4,5),c(5,6,7,8,7),method="spearmanr")
-    x1 = [1, 2, 3, 4, 5]
-    x2 = [5, 6, 7, 8, 7]
-    expected = (0.82078268166812329, 0.088587005313543798)
-    res = stats.spearmanr(x1, x2)
-    assert_approx_equal(res[0], expected[0])
-    assert_approx_equal(res[1], expected[1])
-
-    attributes = ('correlation', 'pvalue')
```

Comment:
did you add this test back? i did not see it

Reply:
So this was the test that appears to be duplicated (one of a handful), but now I cannot seem to find, maybe my mind was playing tricks on me yesterday :).  I added back in the test.